content/influxdb/cloud-dedicated/primers/api/v2/_index.md

```diff
@@ -12,14 +13,289 @@ menu:
 influxdb/cloud-dedicated/tags: [write, line protocol]
 ---
 
-Write data with the InfluxDB v2 API `/api/v2/write` endpoint for InfluxDB Cloud:
-  - Send v2 API requests from client libraries, Telegraf, or your own code.
-    - You can't use the InfluxDB `influx` CLI with InfluxDB Cloud Dedicated.
-    - For help finding the best workflow for your situation, [contact the Engineering team](link).
+Use the InfluxDB v2 API for new workloads and existing v2 workloads that you bring to InfluxDB Cloud Dedicated.
+The InfluxDB Cloud Dedicated v2 `/api/v2/write` endpoint works with token authentication
+and existing InfluxDB 2.x tools and code.
+For help finding the best workflow for your situation, [contact Support](mailto:support@influxdata.com).
+
   <!-- v2 SAMPLE CODE -->
-    - Authentication and authorization
-      - Use a Database Token with the `Authorization: Token` scheme or the `Authorization: Bearer`(???) scheme.
-    - Responses
-  - Use client libraries with the v2 API write endpoint
-  - Use Telegraf with the v2 API write endpoint
-  - Use the v2 API write endpoint
\ No newline at end of file
+
+## Authenticate API requests
+
+  <!--TODO: or the `Authorization: Bearer`(???) scheme -->
```

Comment:
I know it supports `Token`, but I don't know if it supports `Bearer`. AFAIK, `Bearer` auth was only ever added to OSS. I'm not sure who the best person to ask about this would be.

@gaughen Do you happen to know if dedicated supports the `Bearer` authorization scheme?

Reply:
> I know it supports Token, but I don't know if it supports Bearer. 

Yeah, I left the comment b/c I thought I heard or saw someone mention it and it's... more standard... than `Token` so I thought it might be preferred if implemented. That's a lot of "if"s....

Reply:
I confirmed with @wiedld (who has been working on v1 and v2 write for CST) that Bearer is supported.